cyclops-ctrl/internal/cluster/k8sclient/client.go

```diff
@@ -260,6 +260,10 @@ func (k *KubernetesClient) GetManifest(group, version, kind, name, namespace str
 		return "", err
 	}
 
+	if includeManagedFields {
+		resource.SetManagedFields(nil)
+	}
+
```

Comment:
I left a comment about checking if the flag is equal to "true" instead of "false" in `cyclops-ctrl/internal/controller/modules.go` so could you change this to
```go
if !includeManagedFields {
  resource.SetManagedFields(nil)
}
```